### PR TITLE
Tutorial explorer: Student Experience selector in header

### PR DESCRIPTION
**Pull request by breville**
Student Experience is now shown in the header, at desktop widths, as a horizontal selector.  A common component is used for this and the existing Grade selector that's already in the header.

When on mobile layout, the Student Experience filter group uses radio buttons.

Teacher Experience is removed as a filter group.

#### Minimal desktop width (820px):

screenshot 2017-11-03 10 36 43

#### Mobile:
screenshot 2017-11-03 10 37 21

#### Wide desktop width:
screenshot 2017-11-03 10 37 37

### CODE REVIEW

File: apps/src/tutorialExplorer/filterGroupHeaderSelection.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  itemStyle(index) {
+    const value = this.props.selection[0];
+    const selectedIndex = this.props.filterGroup.entries.findIndex(item => (item.name === value));
+
+    // When we have two unselected items next to each other, we want to draw a grey
+    // vertical divider between them, and that's done by rendering a border-left
+    // of the item on the right.
+
+    if (index === selectedIndex) {
```
Comment (Erin007) [inline apps/src/tutorialExplorer/filterGroupHeaderSelection.jsx:68]: This seems complicated, but I don't know of a better way to do it. Consolidate both of the conditions that `return {}` ? I can see how it is more clear what's happening if they are separate. Doesn't need to be changed - just noting it took me a  minute to work through what was happening. Comments helped!

File: apps/src/tutorialExplorer/filterHeader.jsx [hunk 2]
```diff
@@ -17,33 +17,24 @@ const styles = {
     paddingRight: 7,
     backgroundColor: 'white'
   },
-  sticky: {
-    boxShadow: "0 8px 6px -6px #999"
-  },
   barDesktop: {
     color: "dimgrey",
```
Comment (Erin007) [inline apps/src/tutorialExplorer/filterHeader.jsx:21]: Where are these colors coming from? I'm used to seeing something like `color.white` with a color import from utils.

Comment (breville) [reply apps/src/tutorialExplorer/filterHeader.jsx:21]: Good point.  Tutorial Explorer has never used the color import but it should get a separate update to use it everywhere.  Noting.

**Comment by breville** [comment]
FYI just made final updates to this PR.

**Comment by breville** [comment]
circleci failure is known and unrelated.  merging.

**Pull request merged**